code-for-canada/project-thundercat
# Select your language page

# Description

Create a sticky language page selection.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

<img width="986" alt="Screen Shot 2019-06-26 at 4 29 35 PM" src="https://user-images.githubusercontent.com/4640747/60212610-a7f13b00-982f-11e9-9eb5-2023a9dff74e.png">


# Testing

Manual steps to reproduce this functionality:

1.  Go to the home page.
2. Select a language and be taken to the home page in that language.
3. Refresh - you should still have that language selected.

# Checklist
